src/main/frontend/handler/editor.cljs

```diff
@@ -2922,11 +2910,11 @@
                 (state/set-editor-action-data! {:pos (cursor/get-caret-pos input)})
                 (state/set-editor-show-block-commands!))
 
-              (nil? @search-timeout)
```

Comment:
Can't remember why this condition was added 😂 Maybe the intention is to not close the autocomplete when searching for blocks, but it doesn't make sense at all. So thanks for removing this too.